printer_zpl2/README.rst

```diff
@@ -0,0 +1,90 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=====================
+ZPL II Label printing
+=====================
+
+This module extends the **Report to printer** (``base_report_to_printer``)
+module to add a ZPL II label printing feature.
+
+It's done to be used from other modules, or from python code blocks.
```

Comment:
With this statement, I want to explain that the module mainly provides the way to configure labels, a method to print a label (to be called the way you want) and a generic wizard that can be added on any model by a custom module. It doesn't have a full GUI with lots of options.
Maybe my english is not good enough here :D